src/lib.rs

```diff
@@ -1,13 +1,15 @@
 //! This crate contains checked implementations of `transmute()`.
 //!
-//! The functions in this crate are not inherently safe, but just guarded against common simple mistakes
-//! (like trying to create an 8-byte type from 7 bytes).
-//!
-//! Those functions are exactly as safe as the data passed to them - creating a null pointer,
+//! The base functions in this crate are not inherently safe, but just guarded against common simple mistakes
+//! (like trying to create an 8-byte type from 7 bytes.
```

Comment:
🎵 You missed a paren and you liked it 🎵

Reply:
:musical_note: I've got a lovely bunch of parentheses. :musical_note: 

:musical_note: There they are, in a new commit. :musical_note: